tests/unit/auth/v2/queryAuthCheck.js

```diff
@@ -26,3 +27,94 @@ describe('v2: queryAuthCheck', () => {
         }
     }));
 });
+
+describe('v2: queryAuthCheck', () => {
+    let clock;
+
+    beforeEach(() => {
+        clock = sinon.useFakeTimers();
+    });
+    afterEach(() => {
+        process.env.PRE_SIGN_URL_EXPIRY = 604800000;
```

Comment:
I think I missed it in your original PR, could you add the default expiry value to  `/arsenal/lib/constants.js` and use that in `queryAuthCheck.js` and here?  You can have the change in this PR I guess!

Reply:
If you are going to be adding this, *optionally* you might consider changing the default unit to seconds and convert it to the required unit, it's a more familiar unit and less 0s to count :wink:

Reply:
Only do this if it's an easy change I don't want you to break all the stack for this :)

Reply:
Sure, will try that and see if it break anything, so far it doesn't seem like it would, but I'll double-check. :)

Reply:
IMHO, you should, makes it easier to set!